main.tf

```diff
@@ -28,6 +28,9 @@ module "tailscale_subnet_router" {
   instance_type  = var.instance_type
   instance_count = var.instance_count
 
+  monitoring_enabled = var.monitoring_enabled
```

Comment:
@mheffner this changes look good, but can you please run `terraform fmt` on this and then add / commit to fix these alignment issues?

Reply:
:facepalm: